mmdet/models/detectors/rpn.py

```diff
@@ -138,17 +138,6 @@ def aug_test(self, imgs, img_metas, rescale=False):
                                                 flip_direction)
         return [proposal.cpu().numpy() for proposal in proposal_list]
 
-    def show_result(self, data, result, dataset=None, top_k=20):
-        """Show RPN proposals on the image.
-
-        Although we assume batch size is 1, this method supports arbitrary
-        batch size.
-        """
-        img_tensor = data['img'][0]
-        img_metas = data['img_metas'][0].data[0]
-        imgs = tensor2imgs(img_tensor, **img_metas[0]['img_norm_cfg'])
-        assert len(imgs) == len(img_metas)
-        for img, img_meta in zip(imgs, img_metas):
-            h, w, _ = img_meta['img_shape']
-            img_show = img[:h, :w, :]
-            mmcv.imshow_bboxes(img_show, result, top_k=top_k)
+    def show_result(self, data, result, dataset=None, top_k=20, **kwargs):
+        """Show RPN proposals on the image."""
```

Comment:
Why do not we need convert tensor images to numpy anymore? We should add comments to notify that.